modules/json_form_widget/src/StringHelper.php

```diff
@@ -77,6 +77,9 @@ public function handleStringElement($definition, $data, $object_schema = FALSE)
     // Add options if element type is select.
     if ($element['#type'] === 'select') {
       $element['#options'] = $this->getSelectOptions($property);
+      if ($property->empty_value) {
+        $element['#empty_value'] = '';
```

Comment:
Looks like we're going a different direction anyway, but not sure where `$property->empty_value` is supposed to be getting set?